spaceship.zsh

```diff
@@ -1027,7 +1027,7 @@ spaceship_ember() {
 spaceship_kubecontext() {
   [[ $SPACESHIP_KUBECONTEXT_SHOW == false ]] && return
 
-  local kube_context=$(awk -F' *: *' '$1 == "current-context" {print $2}' ~/.kube/config)
+  local kube_context=$([[ -f ~/.kube/config ]] && awk -F' *: *' '$1 == "current-context" {print $2}' ~/.kube/config)
```

Comment:
I think we should check for file before variable definition and exit like in other sections.

https://github.com/denysdovhan/spaceship-zsh-theme/blob/68068b61e5e8852dcdfff1f19a796758f86d2cde/spaceship.zsh#L651-L654